package.json

```diff
@@ -9,10 +9,11 @@
   ],
   "homepage": "https://github.com/mobify/redux-runtypes-schema",
   "version": "0.1.1",
+  "license": "SEE LICENSE IN LICENSE",
   "files": [
     "lib"
   ],
-  "license": "SEE LICENSE IN LICENSE",
+  "main": "lib/index.js",
```

Comment:
This is needed so we can `import createSchemaReducer from 'redux-runtypes-schema'` without needing to also specify `lib/...`

Reply:
👍